src/EA.Weee.Web/Areas/Scheme/Controllers/ManageEvidenceNotesController.cs

```diff
@@ -55,6 +57,63 @@ public async Task<ActionResult> Index(Guid organisationId, ManageEvidenceNotesDi
             }
         }
 
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Index(Guid organisationId)
+        {
+            return RedirectToAction("TransferEvidenceNote", "ManageEvidenceNotes", new { area = "Scheme", organisationId });
+        }
+
+        [HttpGet]
+        public async Task<ActionResult> TransferEvidenceNote(Guid organisationId)
```

Comment:
unit tests. They will be easier once the logic of creating the selected categories is moved to the request creator.

Reply:
will be added in a different branch